client_dashboard/common/build.gradle.kts

```diff
@@ -20,6 +20,18 @@ kotlin {
                 api(libs.compose.runtime)
                 api(libs.compose.foundation)
                 api(libs.compose.material3)
+
+                implementation(libs.voyager.navigator)
+                implementation(libs.voyager.bottomSheetNavigator)
+                implementation(libs.voyager.tabNavigator)
+                implementation(libs.voyager.transitions)
+                implementation(libs.voyager.androidx)
+                implementation(libs.voyager.koin)
```

Comment:
what if we can use a single implementation to implement all these libraries?

check this comment :
https://github.com/TheChance101/beep-beep/pull/107/commits/7e012476ad00b266c3a0bca84ba9c2fedeed8c3c#r1294093672

Reply:
one question are we really need all these libraries ?